CHANGELOG.md

```diff
@@ -33,6 +33,7 @@ or `/query` HTTP endpoints.
 1. [20894](https://github.com/influxdata/influxdb/pull/20894): Display task IDs in the UI.
 1. [21046](https://github.com/influxdata/influxdb/pull/21046): Write to standard out when `--output-path -` is passed to `influxd inspect export-lp`.
 1. [21006](https://github.com/influxdata/influxdb/pull/21006): Add `-p, --profilers` flag to `influx query` command.
+1. [21090](https://github.com/influxdata/influxdb/pull/21090): Synchronize ui folder with ui repository.
```

Comment:
```suggestion
1. [21090](https://github.com/influxdata/influxdb/pull/21090): Update UI to match InfluxDB Cloud.
```
Make the implications more clear for end-users

Reply:
Good suggestion! Done.